### PR TITLE
serve 0.5.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by dawidd6** [comment]
We can remove `GOPATH` stuff btw.

**Comment by chenrui333** [comment]
> We can remove `GOPATH` stuff btw.

good call, just removed it.

File: Formula/serve.rb [hunk 1]
```diff
@@ -16,14 +16,7 @@ class Serve < Formula
...
   def install
-    ENV["GOPATH"] = buildpath
-
-    src = buildpath/"src/github.com/syntaqx/serve"
-    src.install buildpath.children
-    src.cd do
-      system "go", "build", "-ldflags", "-X main.version=#{version}", "-o", bin/"serve", "./cmd/serve"
-      prefix.install_metafiles
-    end
+    system "go", "build", "-ldflags", "-X main.version=#{version}", "-o", bin/"serve", "./cmd/serve"
```
Comment (alebcay) [inline Formula/serve.rb:19]: `system "go", "build", "-ldflags", "-X main.version=#{version}", *std_go_args, "./cmd/serve"`

Comment (chenrui333) [reply Formula/serve.rb:19]: 👍

**Pull request closed without merging**